### PR TITLE
making sure containers fall without user input

### PR DESCRIPTION
**Pull request by NataliaLKB**
I thought it would be best to make it so that regardless of user input, the containers would still fall away.

### CODE REVIEW

**Comment by NataliaLKB** [comment]
@Reettaphant please review

File: script.js [hunk 1]
```diff
@@ -21,22 +21,29 @@
     function stopShaking (element) {
 
         setTimeout(function () {
-            var newClasses = element.className.replace(/\bshake-front-container\b/,'front-container-fall');
-            var child = element.getElementsByClassName('fc-item')[0];
+            var newClasses  = element.className.replace(/\bshake-front-container\b/,'front-container-fall');
+            var child       = element.getElementsByClassName('fc-item')[0];
```
Comment (Reettaphant) [inline script.js]: The child can variable can be removed.

Comment (NataliaLKB) [inline script.js]: Done please let me know what else to change, or give a :+1:

**Comment by Reettaphant** [comment]
Looks good. :+1:

**Pull request merged**